BLOG.adoc

```diff
@@ -134,7 +144,7 @@ Since we are running the https://grafana.com/grafana/[Grafana] stack with https:
         </http>
         <format>
             <label>
-                <pattern>app=${appName},host=${HOSTNAME},traceID=%X{traceId:-NONE},level=%level</pattern>
+                <pattern>app=${appName},host=${HOSTNAME},level=%level,traceId=%X{traceId:-00000000000000000000000000000000}</pattern>
```

Comment:
As far as I remember labels are indexed so the `traceId` here can overwhelm loki.

Reply:
So what should we do?

Reply:
Remove it from here and tell loki how to extract it from the message:
See: https://github.com/jonatan-ivanov/teahouse/commit/94bfb81623a61cc71be870e96ff4785407420c89

I guess extracting from the message is already happening, so, I guess it is just removing from the logback config:
https://github.com/jonatan-ivanov/teahouse/blob/5b1c2d5e04c9ac7d6cbd2123eae3279ff6fae369/docker/grafana/provisioning/datasources/datasource.yml#L58-L62